vector/src/androidTest/java/im/vector/app/ui/robot/ElementRobot.kt

```diff
@@ -66,6 +66,38 @@ class ElementRobot {
         waitForHome()
     }
 
+    // Used for michaelk testing
+    fun login(homeserverURL: String, userId: String, password: String) {
```

Comment:
if we expand the Traffic light suite I can imagine a dedicated `TrafficLightRobot`